docs/quickstarts/packet.md

```diff
@@ -150,90 +150,89 @@ lokomotive-demo-pool-1-worker-1    True     KubeletReady    kubelet is posting r
-   ssh-agent](https://help.github.com/en/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent#adding-your-ssh-key-to-the-ssh-agent).
-2. Retry [Step 5](#step-5-create-lokomotive-cluster).
+- Verify the correct private SSH key was added to `ssh-agent`.
+- Verify that you can SSH into the created controller nodes from the machine running `lokoctl`.
 
 ### Packet provisioning failed
 
-For failed machine provisioning on Packet end, retry [Step 5](#step-5-create-lokomotive-cluster).
+Sometimes the provisioning of servers on Packet may fail. In this case, retrying the deployment by
+re-running `lokoctl cluster apply` may help.
 
-### Insufficient availability of nodes types on Packet
+### Insufficient capacity on Packet
 
-In the event of failed Packet provisioning due to machines of type `controller_type` or
-`workers_type` not available.  You can check the Packet API [capacity
-endpoint](https://www.packet.com/developers/api/capacity/) to get the current capacity and decide on
-changing the facility or the machine type.
+Sometimes there may not be enough hardware available at a given Packet facility for a given machine
+type. In this case, either select a different node type and/or Packet facility, or wait a while for
+more capacity to become available. You can check the current capacity status on the Packet
+[API](https://www.packet.com/developers/api/capacity/).
 
 ### Permission issues
 
-  * If the failure is due to insufficient permissions on Packet, check the permission on the Packet
-    console.
-  * This generally happens if user is using `Project Level API Key` and not `User Level API Key`.
-
-### Failed installation of components that require disk storage
-
-For components that require disk storage such as [Openebs storage
-class](../configuration-reference/components/openebs-storage-class.md), [Prometheus
-Operator](../configuration-reference/components/prometheus-operator.md) machine types with spare disks
-should be used.
+If the deployment fails due to insufficient permissions on Packet, verify your Packet API key has
+permissions to the right Packet project.
 
-## Conclusion
-
-After walking through this guide, you've learned how to set up a Lokomotive cluster on Packet.
+If the deployment fails due to insufficient permissions on AWS, ensure the IAM user associated with
+the AWS API credentials has permissions to create records on Route 53.
 
 ## Next steps
 
-You can now start deploying your workloads on the cluster.
-
-For more information on installing supported Lokomotive components, you can visit the [component
-configuration references](../configuration-reference/components).
+**Lokomotive components** complement the "stock" Kubernetes functionality by adding features such
+as load balancing, persistent storage and monitoring to a cluster. Refer to
+[this](../concepts/components.md) document to learn about the available components as well as how
+to deploy them.
```

Comment:
Yes, I've been contemplating the same myself, and decided to err on the "short and concise" side. I'll add a sample app. The question is - do I also add MetalLB and Contour? This significantly increases the complexity of the guide (allocate an EIP, wait for the request to get approved, specify the EIP in the config, enable BGP, ...).

Reply:
No, sorry, I didn't want the guide to add an app deployment. Please don't.

I think you got it from the part I mention that a linked tutorial does expose an app (using service load balancer)? Is this correct? I was just pointing out that the tutorial linked **does** deploy an app and that won't work in the created cluster. If we are linking to a tutorial that can't be completed in this cluster, at least we should be clear about it. That was the intent of that part of my message.

Reply:
Fair enough. What should I change then?

Reply:
Well, I was asking for your opinion. You should start by saying what you think :)

As I said:
> I think I lean towards the latter. To have the quickstart just be a cluster and on bare-metal, to match the features other platform have "out of the box", you have to do more work (just add a few components and config). It might make sense to add another guide, as part of another PR, with steps to deploy a cluster that feature-wise is comparable to AWS quickstart (load balancer and ingress controllers, at least).

Therefore, if we decide to go down that route, I think we should state here the limitations this create cluster has, create an issue about explaining how to have an *useful* cluster in packet (i.e. that can receive traffic, at the very least).

Maybe, another way to solve it, can be: state the limitations here and just link to the components needed to receive traffic in the cluster.

But if you prefer to go some other route (like add MetalLB here, just keep it as it is for some reason I don't know, etc.), it is another option too. The idea was to discuss this issue, see the inputs from others and have concensus on what to do, not dictate anything :)

Reply:
We've settled on demonstrating a sample app using port forwarding and linking to an LB + ingress guide in the "next steps" section.